learn/components/styling.md

```diff
@@ -3,4 +3,191 @@ title: Styling
 order: 5
 ---
 
-{% stub %}
+## (Introduction)
+
+- Web Components have powerful styling capabilities.
+  - _Let’s start off positive, even if it’s going to get a little
+    hairy._
+- Shadow DOM encapsulation makes Web Component styles generally easier
+  to author.
+  - Elements within a shadow tree cannot be directly selected by styles
+    declared outside of it. Conversely, elements outside of a shadow
+    tree cannot be directly selected by styles declared within it.
+  - Web Component authors need not worry about naming conflicts for CSS
+    classes outside of a shadow tree.
+  - Web Component authors can use less specific selectors without much
+    worry as there are generally less elements used within a shadow
+    tree.
+- Encapsulation is not isolation. Styles set outside of a shadow tree
+  can indirectly influence styles within a shadow tree.
+  - _Use this to transition to the next section..._
+
+## Shadow Hosts and CSS Inheritance
+
+It is very important to understand the role of a **shadow host**.
+
+### What is a Shadow Host
```

Comment:
I feel like this is the kind of topic we want to pull out into its own page. I think the “shadow dom” section will likely span multiple pages.

Reply:
For sure, as I was writing all this, I wondered to myself if I should have gotten started on the Shadow DOM page instead.